### ISSUE TITLE
DL3006 on scratch image.

### ISSUE THREAD

**Issue by pakraticus** [lukasmartinelli/hadolint#85]
Expect no warning on typical docker multi-stage use case

```Dockerfile
FROM alpine:3.6 AS build
RUN buildstuff

FROM scratch
COPY --from=build /result/ /result
ENTRYPOINT [ /result/bin/tool ]
MAINTAINER ...
```

Instead receive warning `DL3006 Always tag the version of an image explicitly` against the `FROM scratch` line.
Cannot use `scratch:0` as scratch is a reserved internal name.

Note this also provides a false warning some base image Docker files with

```Dockerfile
FROM scratch
ADD tarball
....
```

**Comment by zemanlx** [comment]
@pakraticus this issue has been fixed a year ago in https://github.com/lukasmartinelli/hadolint/commit/f951912765abe12e70372a56fde568c5e252febc. What version do you use?

**Comment by zemanlx** [comment]
No response, closing.
